torch/csrc/api/include/torch/nn/modules/named_any.h

```diff
@@ -37,48 +37,50 @@ namespace nn {
 ///     }
 ///   };
 ///
-///   Sequential sequential(modules_ordered_dict({
-///     {"m1", std::make_shared<M>(1)},  // shared pointer to `Module` is supported
-///     {std::string("m2"), M(2)},  // `Module` is supported
+///   AnySequential sequential(modules_ordered_dict({
+///     {"m1", std::make_shared<M>(1)},  // shared pointer to `Module` is
+///     supported {std::string("m2"), M(2)},  // `Module` is supported
```

Comment:
I think "supported" should go to the previous line, so that the named submodule braced-init lists are aligned.